### PR TITLE
Community tab top banner section changes

### PR DESCRIPTION
**Pull request by aggarwalkartik555**

### CODE REVIEW

File: src/components/images/images.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ import blog2 from "../../assets/jpg/blog2.jpg";
 import coursesSignupBtn from '../../assets/svg/courses-signup-btn.svg';
 import coursesBanner from '../../assets/svg/courses-banner.svg';
 import coachingInputBtn from '../../assets/svg/coaching-input-btn.svg';
+import searchIcon from '../../assets/svg/search-icon.svg';
```
Comment (vigilantee) [inline src/components/images/images.js:20]: check if svg's size is less than the jpg? if not, please use jpg.

File: src/components/community/community.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                font-family: Inter;
+                font-size: 48px;
+                font-style: normal;
+                font-weight: 700;
+                line-height: normal;
+            }
+            .community-desc{
+                color: var(--foundation-black-black-800, #2C3032);
+                text-align: center;
+                font-family: Inter;
```
Comment (vigilantee) [inline src/components/community/community.css:27]: do we have this font in bundle? If not please ping me.

File: src/components/community/community.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                        font-weight: 600;
+                        line-height: normal;
+                    }
+                }
+            }
+        }
+    }    
+}
+
+@media only screen and (max-width: 767px) {
```
Comment (vigilantee) [inline src/components/community/community.css:107]: is this '767' consistent through out the website or have we used 768 as well.
If yes please make it consistent.

File: src/components/images/images.js (hunk 1 above)
Comment (aggarwalkartik555) [reply src/components/images/images.js:20]: checked this, svg size is less than jpg

File: src/components/community/community.css (hunk 3 above)
Comment (aggarwalkartik555) [reply src/components/community/community.css:107]: It's done

**Pull request merged**